nix/overlays/postgresql-legacy.nix

```diff
@@ -0,0 +1,20 @@
+self: super:
+# Overlay that adds legacy versions of PostgreSQL that are supported by
+# PostgREST.
+{
+  # PostgreSQL 9.5 was removed from Nixpkgs with
+  # https://github.com/NixOS/nixpkgs/commit/72ab382fb6b729b0d654f2c03f5eb25b39f11fbb
+  # We pin its parent commit to get the last version that was available.
+  postgresql_9_5 =
```

Comment:
Not sure if we need that - we agreed in #1820 to remove 9.5 soon anyway.

This boils down to whether we'll release v8 before merging this here or after.

@steve-chavez WDYT? How close are we do getting v8 ready docs-wise?

Reply:
Suggest that we merge this before, as we'll bake the latest 13.3 Postgres bindings into the static executable that way (not that there is anything critical there, but can't hurt: https://www.postgresql.org/docs/release/13.3/).

In any case, I would also keep this file around, with commented out content if it's not needed right now - we'll need it now and again later.

Reply:
Hey guys, good to see you back!

> This boils down to whether we'll release v8 before merging this here or after.

I don't see a problem with removing 9.5 after v8, so we can merge this.

> How close are we do getting v8 ready docs-wise?

@wolfgangwalther There's still work to be done in the docs. Me and @laurenceisla thought it'd be better to tag/release v8 with just the CHANGELOG in the github release(next week tops, after finishing the last breaking change https://github.com/PostgREST/postgrest/issues/1643) - and once that's done we can take a bit more time with finishing the v8 docs.